client/src/modules/dashboard/Events/components/Actions.tsx

```diff
@@ -72,6 +73,9 @@ const Actions: React.FC<ActionsProps> = ({
       <Button size="sm" colorScheme="blue" onClick={clickEmailAttendees}>
         Email Attendees
       </Button>
+      <ShareEventPop
+        link={`${process.env.CLIENT_LOCATION}/events/${event.id}?emaillink=true`}
```

Comment:
```suggestion
        link={`${process.env.NEXT_PUBLIC_CLIENT_URL}/events/${event.id}?emaillink=true`}
```
Sorry, the env vars are needlessly complicated. I shouldn't have stored `CLIENT_LOCATION` in two variables.

Anyway, this is the one you need.